ground/gcs/src/plugins/boards_dronin/boards_dronin.pro

```diff
@@ -18,7 +18,9 @@ HEADERS += \
     pikoblx.h \
     playuavosd.h \
     seppuku.h \
-    seppukuconfiguration.h
+    seppukuconfiguration.h \
+    sprf3e.h \
```

Comment:
I named it SPRF3E since the FrSky version is named XSRF3E. Since Evo are the only "E"-boards, I see no problem in the short version. FrSky has also an XMPF3E, also with onboard F3 Evo. I do not have this board because of missing telemetry. But It should work like the XSRF3E.